src/doc/DocHighlightThread.js

```diff
@@ -68,12 +68,12 @@ class DocHighlightThread extends AnnotationThread {
      * @return {void}
      */
     destroy() {
+        this.emit(THREAD_EVENT.render, this.location.page);
```

Comment:
Should this still be the `THREAD_EVENT.threadCleanup`?

Reply:
I can rename it to threadCleanup but essentially it should trigger a re-render of the canvas so I changed the event to represent that. I can rename it if there's something else that would make sense?

Reply:
oh no if that's what makes the most sense then that's fine. I was just noticing that before it was threadCleanup event